src/om_css/core.cljc

```diff
@@ -28,9 +23,17 @@
 (defn local-kw
   "Generate a keyword for a localized CSS class for use in Garden CSS syntax as a localized component classname keyword."
   ([comp-class]
-   (keyword (str "." (cssify (fq-component comp-class)))))
+   (str "." (cssify (fq-component comp-class))))
   ([comp-class nm]
-   (keyword (str "." (cssify (fq-component comp-class)) "__" (name nm)))))
+    (str "." (cssify (fq-component comp-class)) "__" (name nm))))
+
+(defn call-css [component]
```

Comment:
Hm.`call-css` is just meant to be the uniform way of accessing the "static" protocol method `css`. Perhaps name this something like `localize-css` and have it take the output of `call-css` (which then requires wrapping it later where `call-css` is used, e.g. in `css-merge`). 

I could be convinced to let this one stand, since it doesn't change any existing behavior (just augments it with new)...it's just a little less than ideal in terms of structure.